### PR TITLE
Add finger_edu robot

### PR DESCRIPTION
**Pull request by jviereck**
Adding the finger robot as discussed in #90 .

I've added unit tests. However, it is not clear to me how to run them. Can someone tell me what I need to do for this?

### CODE REVIEW

**Comment by hrp2-14** [comment]
Hi ! This project doesn't usually accept pull requests on master. If this wasn't intentionnal, you can change the base branch of this pull request to devel (No need to close it for that). Best, a bot.

File: unittest/test_load.py [hunk 1]
```diff
@@ -79,6 +79,9 @@ def test_solo(self):
     def test_solo12(self):
         self.check('solo12', 19, 18)
 
+    def test_solo(self):
```
Comment (jcarpent) [inline unittest/test_load.py:82]: I think the name is not correct here.

**Review by jcarpent — CHANGES_REQUESTED** [review]
Thanks @jviereck for this nice contribution. Slight remark below.

File: unittest/test_load.py (hunk 1 above)
Comment (jviereck) [reply unittest/test_load.py]: Thanks for the quick review! I've corrected the name.

**Review by cmastalli — APPROVED** [review]
You only need to format your code. For that, run `yapf -ri .` in the root of this project.

**Review by jcarpent — APPROVED** [review]
Thanks for the quick fix.

**Comment by jviereck** [comment]
> For that, run yapf -ri . in the root of this project.

If I do this it change the content of a lot of the files in the repo (also ones I didn't work on). Is this intended? Should these changes be included in this PR?

**Comment by jcarpent** [comment]
Just take into account the change related to your current PR.

**Comment by jviereck** [comment]
Okay, I was missing a single newline in terms of formatting. Pushed the change only to that code section of the file as a new commit.

**Review by nim65s — APPROVED** [review]
Thanks !

**Comment by jviereck** [comment]
Thanks everyone for the review. Is there something else missing to merge this? I don't have a green merge button so I guess some maintainer of this package has to do the merge?

**Pull request merged**